src/main/resources/liquibase-changelog/versions/med-1.yaml

```diff
@@ -0,0 +1,83 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: wojtekbier03
+      changes:
+        - createTable:
+            tableName: PATIENTS
+            columns:
+              - column:
+                  autoIncrement: true
+                  constraints:
+                    nullable: false
+                    primaryKey: true
+                    primaryKeyName: pk_patient_id
+                  name: id
+                  type: BIGINT
+              - column:
+                  name: email
+                  type: VARCHAR(255)
+                  constraints:
+                    unique: true
+                    uniqueConstraintName: uk_patient_email
+              - column:
+                  name: id_card_no
+                  type: VARCHAR(255)
+              - column:
+                  name: first_name
+                  type: VARCHAR(255)
+              - column:
+                  name: last_name
+                  type: VARCHAR(255)
```

Comment:
Może takie było wymaganie, ale kolumny first_name/last_name to antypattern:
https://shinesolutions.com/2018/01/08/falsehoods-programmers-believe-about-names-with-examples/